<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>479</width>
    <height>386</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Akıllı Sinyal</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/resim/ecg.jpg);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="txt_time">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>113</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>62</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="text">
     <string>Süre:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_analyze">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>60</y>
      <width>90</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Analiz Et</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_quit">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>100</y>
      <width>90</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Çıkış</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_file">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>300</y>
      <width>101</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Dosya Seç</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_file">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>300</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>479</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="xy.qrc"/>
 </resources>
 <connections/>
</ui>
